### PR TITLE
microshift: bump periodic test openshift-test bin version to 4.12

### PR DESCRIPTION
**Pull request by copejon**
[USHIFT-322](https://issues.redhat.com//browse/USHIFT-322)

### CODE REVIEW

File: ci-operator/config/openshift/microshift/openshift-microshift-main__periodics.yaml [hunk 1]
```diff
@@ -82,6 +120,20 @@ tests:
...
+    env:
+      TEST_SUITE: openshift/conformance/parallel
+    workflow: openshift-microshift-e2e-test
+- as: e2e-openshift-conformance-serial
+  cron: 0 0 * * *
+  steps:
+    cluster_profile: gcp
+    env:
+      TEST_SUITE: openshift/conformance/serial
+    workflow: openshift-microshift-e2e-test
```
Comment (deads2k) [inline ci-operator/config/openshift/microshift/openshift-microshift-main__periodics.yaml:136]: great!  Thanks

**Comment by ingvagabund** [comment]
Unfortunately, there are no e2e logs in either of the conformance e2e tests:
- https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/31651/rehearse-31651-periodic-ci-openshift-microshift-main-periodics-e2e-openshift-conformance-serial/1562804610450591744/artifacts/e2e-openshift-conformance-serial/openshift-microshift-e2e-test/build-log.txt
- https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/31651/rehearse-31651-periodic-ci-openshift-microshift-main-periodics-e2e-openshift-conformance-parallel/1562804610593198080/artifacts/e2e-openshift-conformance-parallel/openshift-microshift-e2e-test/build-log.txt

**Comment by ingvagabund** [comment]
Both jobs are reporting a lot of missing permissions to list pods:
```
W0826 21:18:13.442534     103 loader.go:221] Config not found: /tmp/kubeconfig-975685197
W0826 21:18:13.446223     103 loader.go:221] Config not found: /tmp/kubeconfig-975685197
W0826 21:18:13.446561     103 loader.go:221] Config not found: /tmp/kubeconfig-975685197
W0826 21:18:13.447056     103 loader.go:221] Config not found: /tmp/kubeconfig-975685197
W0826 21:18:13.447678     103 loader.go:221] Config not found: /tmp/kubeconfig-975685197
I0826 21:18:13.448358     103 shared_informer.go:255] Waiting for caches to sync for SimultaneousPodIPController
W0826 21:18:13.448513     103 loader.go:221] Config not found: /tmp/kubeconfig-975685197
W0826 21:18:13.455365     103 reflector.go:324] k8s.io/client-go@v0.24.0/tools/cache/reflector.go:167: failed to list *v1.Pod: pods is forbidden: User "system:serviceaccount:ci-op-fr3fty2k:e2e-openshift-conformance-serial" cannot list resource "pods" in API group "" at the cluster scope
E0826 21:18:13.455422     103 reflector.go:138] k8s.io/client-go@v0.24.0/tools/cache/reflector.go:167: Failed to watch *v1.Pod: failed to list *v1.Pod: pods is forbidden: User "system:serviceaccount:ci-op-fr3fty2k:e2e-openshift-conformance-serial" cannot list resource "pods" in API group "" at the cluster scope
W0826 21:18:13.455773     103 reflector.go:324] k8s.io/client-go@v0.24.0/tools/cache/reflector.go:167: failed to list *v1.Pod: pods is forbidden: User "system:serviceaccount:ci-op-fr3fty2k:e2e-openshift-conformance-serial" cannot list resource "pods" in API group "" at the cluster scope
```

**Comment by ingvagabund** [comment]
Looks good

**Pull request merged**